openfecli/commands/gather.py

```diff
@@ -77,7 +136,9 @@ def get_type(res):
         return 'solvent'
 
 
-def legacy_get_type(res_fn):
+def legacy_get_type(res_fn:os.PathLike|str)->Literal['vacuum','solvent','complex']:
+    """TODO: when to deprecate this?"""
```

Comment:
I think once get_type gets updated to not rely on the key names then we can drop this.